process/TAC-Decision-Process.md

```diff
@@ -0,0 +1,43 @@
+
+Broadly, there are three main methods that the TAC will use to make decisions and ensure those decisions are appropriately documented:
+
+## 1.) Simple and/or non-consequential changes ##
+For asynchronous work items that do not have long-term/strategic consequences, the TAC will use lazy consensus to determine whether to adopt the proposal and ensure that it will be properly documented.  Issues and PRs should be created within the TAC GitHub repository and assigned to the TAC group for review.  Provided that at least two TAC members agree, support, and approve the suggested change, there are no objections raised by any TAC members, and no less than one week has transpired since the TAC was notified of the change, the proposal will be merged and adopted.
+
+This includes such things as documentation updates/minor changes
+
+
+## 2.) For work items discussed at the periodic TAC calls ##
+The TAC must have quorum (majority of members present) and a simple majority of the members present must approve of a proposal or topic.  This decision will be captured in the TAC meeting notes as well as an issue within the TAC repository. 
+
+
+
+## 3.) For  major items that require a formal vote ##
+An issue or PR must be created within the TAC repository.  TAC members will be asked via the TAC mailing list to review, comment, and decline/approve the item within the GitHub issue/PR.  The TAC will be given 10 business days to review and consider the request.  These topics ideally should also be presented at a TAC meeting and publicly discussed/debated prior to the closing of voting.  
+
+This includes items such as changes to the TAC technical vision, Foundation strategy, WG/project adoption, funding requests, adopting new Working Group, etc.).
+
+# Pull request types #
+
+Every PR must be categorized using one of the following <type> values. The purpose is twofold: to make it easier for readers to understand the scope of the PR at a glance, and to allow us to adjust the minimum review period and number of approvers based on how sensitive the PR is.
+
+Use the closest entry in the table that applies, selecting the first one if multiple apply. If you are not sure which type to use, take a guess and a maintainer will update if needed.
+
+|  Type  | Meaning | Waiting period |	# Approvers |
+| ------ | ------- | -------------- | ----------- |
+| Major  | Proposed changes to Charter, Technical Strategy, or other cross-foundation changing proposal.  If approved, than moved to the Governing committee for review, move to GB. | 15d | 7 |
+| TI lifecycle | A new TI or updated TI lifecycle stage. | 10d | 5 |
```

Comment:
My vote here would be to bump this up to 7.

Reply:
done!